Cleaning RB issues

hernanmd opened the pull request:
- Clean #renameClassVariable:to:in:around: which has no users: https://github.com/pharo-project/pharo/issues/11512
- Clean RBReplacePreview without users in Pharo 11: https://github.com/pharo-project/pharo/issues/11515 
- Enabled the copy package command when right-click over a selected package in a Calypso browser. The SycCopyPackageCommand has a commented in the menu item enabler in its #canBeExecutedInContext:, and it was always returning false.

Commit:
Clean #renameClassVariable:to:in:around: which has no users: https://github.com/pharo-project/pharo/issues/11512
Clean RBReplacePreview without users in Pharo 11: https://github.com/pharo-project/pharo/issues/11515
Enabled the copy package command when right-click over a selected package in a Calypso browser. The SycCopyPackageCommand has a commented in the menu item enabler in its #canBeExecutedInContext:, and it was always returning false.

Commit:
Clean #renameClassVariable:to:in:around: which has no users: https://github.com/pharo-project/pharo/issues/11512
Clean RBReplacePreview without users in Pharo 11: https://github.com/pharo-project/pharo/issues/11515
Enabled the copy package command when right-click over a selected package in a Calypso browser. The SycCopyPackageCommand has a commented in the menu item enabler in its #canBeExecutedInContext:, and it was always returning false.

Ducasse commented on src/Refactoring-Core/RBNamespace.class.st:
```diff
@@ -667,15 +667,6 @@ RBNamespace >> renameClass: aRBClass to: aSymbol around: aBlock [
 	^ change
 ]
 
-{ #category : #'private - changes' }
```
Why do you remove it?

hernanmd replied:
Because you wanted to #11512 :)

Ducasse commented:
I think that balsa wrote a better preview so we can remove this one indeed.
For the copy this is cool. 
I would like to have the rename with regexp available.

Commit:
Merge 5ea2244c14d49172db03535d7281ed8815dcd2d7

Commit:
Merge branch 'pharo-project:Pharo11' into iss-rb-cleaning

Commit:
Add a new Copy classes refactoring UI to enable customization of copied class names based on regex pattern.

Commit:
Merge 6d5a01df42821086a4bbbaef146dcf572a86aba2

MarcusDenker commented:
- [CI] System variable environment on windows does not use adequate encodings #11407

Commit:
Merge branch 'pharo-project:Pharo11' into iss-rb-cleaning

Commit:
Better class selection for classes to be copied using UI with support of regexes (SycCopyClassUsingPatternPresenter)
As there could be many classes to copy, inform user during copying using job run.
Add tests.

Commit:
Better class selection for classes to be copied using UI with support of regexes (SycCopyClassUsingPatternPresenter)
As there could be many classes to copy, inform user during copying using job run.
Add tests.

Commit:
Cleaning and use better class name.

Ducasse commented:
This is nice. Did you see the Driver concept? so that we do not add interaction in the Commander nor the refactoring?

Ducasse commented:
I could not see the results of the tests :(

hernanmd commented:
> I could not see the results of the tests :(

In the RBCopyPackageParametrizedTest #tearDown I remove the package created for test.

```smalltalk
tearDown

	(RBRefactoryChangeFactory instance removePackageNamed: 'RefactoringSmallPackageForTest')
		execute.
	super tearDown
```

I'll delete it for the next commit

hernanmd commented:
> I could not see the results of the tests :(

I didn't see it, some link would be useful.

Commit:
Fix teadDown in RBCopyPackageParametrizedTest
Re-implements https://github.com/pharo-project/pharo/issues/11524

Ducasse commented:
I meant in the CI. Apparently it was doing a 404 or something like that.

Ducasse commented:
It seems to work now. just super slow.

Ducasse commented:
osx-64 / Tests-osx-64 / testAndMakeSureSuperTearDownIsCalledAsLastMessageInTearDownMethodsOfTestCases – MacOSX64.ReleaseTests.CleanCode.ProperlyImplementedSUnitClassesTest
<1s
unix-64 / Tests-unix-64 / testAndMakeSureSuperTearDownIsCalledAsLastMessageInTearDownMethodsOfTestCases – Unix64.ReleaseTests.CleanCode.ProperlyImplementedSUnitClassesTest
<1s
Existing failures - 8
windows-64 / Tests-windows-64 / testAndMakeSureSuperTearDownIsCalledAsLastMessageInTearDownMethodsOfTestCases – Windows64.ReleaseTests.CleanCode.ProperlyImplementedSUnitClassesTest

Ducasse commented on src/Refactoring2-Transformations-Tests/RBCopyPackageParametrizedTest.class.st:
```diff
@@ -16,59 +16,102 @@ RBCopyPackageParametrizedTest >> constructor [
+{ #category : #running }
+RBCopyPackageParametrizedTest >> tearDown [
+
+	super tearDown.
```
super tearDown should be the last expression.

hernanmd replied:
Ok, fix in https://github.com/pharo-project/pharo/commit/e2dc9b8466c9745af2f71b97a859c170334b27fb

Commit:
Merge e2dc9b8466c9745af2f71b97a859c170334b27fb
